my-dsl.md

```diff
@@ -2,37 +2,105 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+The language is called [ACSL](http://frama-c.com/acsl.html), but the tools to
+work with it are collectively [Frama-C](http://frama-c.com/what_is.html) and a
+variety of [plugins](http://frama-c.com/plugins.html).
 
 # Domain
 _Describe the language's domain in five words._
 
+Formal specifications for C code
+
 
 # Computational model
 _What is the underlying computational model of this language? To answer this 
 question, provide a high-level description (no more than 100 words) of the 
 computation that occurs when someone executes a program in this language._
 
+ACSL is not exactly an executable language, _per se_, and what happens depends
+very much on the plugins used. It is a specification language, which can be used
+to add assert statements, prove functions correct, prove loops terminate, or
+other things. This is done by a Frama-C plugin or plugins reading the
+declarative specifications to figure out what it needs to do (e.g., what
+assertions to add, what test cases to prove, etc.), and then attempting to do
+it (e.g., output new C code with additional assert statements, give cases to a
+theorem prover, etc.).
+
```

Comment:
We kind-of agreed in class on Monday that a PL and thus a DSL had to be executable to some extent, otherwise it would just be a language and not a programming language. It appears that ACSL is somewhat executable, but a better description about _how_ it executes would answer this question more nicely